helm/cloud-provider-vsphere/Chart.yaml

```diff
@@ -13,18 +20,3 @@ annotations:
 maintainers:
   - name: Team Rocket
     email: team-rocket@giantswarm.io
```

Comment:
Quite a few changes in this file, but that's because the chart.yaml used to be a general file for the whole bundle of apps whereas now it's only relevant for the CPI.